releasedir/fs_release_dir.go

```diff
@@ -206,6 +206,10 @@ func (d FSReleaseDir) BuildRelease(name string, version semver.Version, force bo
 		return nil, err
 	}
 
+	if d.blobsDir.ContainsSymlinks() {
+		return nil, bosherr.Error("Execute `bosh-release` command to clean up any blobs artifacs left from using the bosh-cli v1.")
```

Comment:
Can this message be fixed and cleaned up and perhaps suggest why it looks wrong? Perhaps

    Bailing because symlinks found in blobs directory. If switching from CLI v1, please use the `reset-release` command.